api/color.ts

```diff
@@ -0,0 +1,45 @@
+import { PrismaClient } from '@prisma/client'
+import { VercelRequest, VercelResponse } from '@vercel/node'
+import Vibrant from 'node-vibrant';
+
+const QUALITY = 2;
+
+const prisma = new PrismaClient();
+
+export default async function handler(_: VercelRequest, response: VercelResponse) {
+    var result = await prisma.albums.findMany({
+        where: {
+            vibrant_color: null,
+            image_url: {
+
```

Comment:
This empty object, does that mean it's not present or something?

Reply:
To be honest, I have no recollection of this :upside_down_face:

Reply:
I think vibrant_color: null should be enough